templates/mobile/developers.html

```diff
@@ -0,0 +1,27 @@
+{% extends "mobile/base_mobile.html" %}
+
+{% block title %}Ubuntu for phones and tablets{% endblock %}
```

Comment:
I imagine you will update the titles and descriptions when the content is available. Might be an idea to remove the content for now so its not missed when going live.